### PR TITLE
Upgrade the logging dependencies to the latest

### PR DESCRIPTION
**Pull request by epidemia**
Implemented #405.

### CODE REVIEW

File: strongbox-parent/pom.xml [hunk 1]
```diff
@@ -68,13 +68,13 @@
         <version.httpclient>4.5</version.httpclient>
         <version.jackson>2.8.7</version.jackson>
         <version.javaee>7.0</version.javaee>
-        <version.jcl>1.7.24</version.jcl>
+        <version.jcl>1.8.0-beta0</version.jcl>
```
Comment (carlspring) [inline strongbox-parent/pom.xml]: Could this be set to the latest known official (non-beta) release, please?

**Comment by carlspring** [comment]
Hi!

Thank you for your contribution!

Would you mind signing the ICLA, as described in the [Contributing](https://github.com/strongbox/strongbox/blob/master/CONTRIBUTING.md) page?

Kind regards,

Martin

File: strongbox-parent/pom.xml (hunk 1 above)
Comment (epidemia) [reply strongbox-parent/pom.xml]: Yes, it is.

**Comment by carlspring** [comment]
Great! Could you also please sign the PDF and bounce it back to me by e-mail?

**Comment by epidemia** [comment]
Done

**Comment by carlspring** [comment]
Thank you very much! :)

**Comment by carlspring** [comment]
Would you like to [join our chat](gitter.im/strongbox/strongbox)? Are there maybe any other tasks you'd like to pick?

**Comment by epidemia** [comment]
I joined the chat. I will take several tasks into the work.

**Pull request merged**